### PR TITLE
inc-sync before home page rendering

### PR DESCRIPTION
**Pull request by cindytung**

### CODE REVIEW

File: cal/cal/views.py [hunk 1]
```diff
@@ -10,13 +10,20 @@
 from oauth2client.client import OAuth2WebServerFlow, AccessTokenCredentials
 from oauth2client.django_orm import Storage
 
+from rest_framework.decorators import api_view
+
 from social.apps.django_app.utils import psa
 
 
+@api_view(('GET',))
```
Comment (MaxFangX) [inline cal/cal/views.py]: Are you sure you need this? Doesn't look like it serves any purpose

Comment (cindytung) [reply cal/cal/views.py]: I get a 'WSGIRequest' object has no attribute 'query_params' error without it

Comment (MaxFangX) [reply cal/cal/views.py]: You can use `request.GET.get('no_sync')`. We shouldn't mix in django REST framework in the pure Django part of our app.

**Review by MaxFangX — APPROVED** [review]

**Comment by MaxFangX** [comment]
+2

**Pull request merged**